Automattic/wp-calypso
# Add initial launchpad checklist creation logic

#### Proposed Changes

* This PR adds styling for the Sidebar and logic for creating + displaying checklist items.

Disabled completed tasks.
Removed circle icon that was displayed when checklist item is incomplete.
Removed arrow icon that was displayed when checklist item is complete.
Added logic that uses `flow` param to select the correct list of checklist items to display on UI.
Added initial logic for creating/updating checklist item objects.
Created hook to retrieve `flow` param from the current site URL.
The 'Write your first post' link will navigate to `/post/` and the 'Add links` will navigate to `/site-editor`.

#### Testing Instructions



* Check out PR 
* Navigate to http://calypso.localhost:3000/setup/launchpad?flow=newsletter&siteSlug=[SITE_SLUG_HERE]
* You should see the following checklist: 

<img width="490" alt="image" src="https://user-images.githubusercontent.com/10482592/185681403-41beb652-491d-489c-bf30-9958bd9d0229.png">

You can compare to the current Figma designs: 9wpoYJrMlYKAKUYvVBMN74-fi-160%3A54215


#### Pre-merge Checklist






Related to #66704, #66125
